dprox.js

```diff
@@ -43,12 +43,16 @@ function makeProxy(host, port, hosts) {
 			if(host.preservePrefix) {
 				options.proxyReqPathResolver = req => req.originalUrl;
 			}
-			if(requestHeaders) {
-				options.proxyReqOptDecorator = proxyReqOptions => {
+			options.proxyReqOptDecorator = proxyReqOptions => {
+				if(host.ignoreInsecure) {
+					proxyReqOptions.rejectUnauthorized = false;
```

Comment:
As I understand it (not entirely sure though), this only deactivates certificate validation - so wouldn't it be more correct to call this option `ignoreCertificate`?

Reply:
I’ve been waffling regarding the name as well.

I understand the option the same way. [From the documentation](https://nodejs.org/docs/latest/api/tls.html#tls_tls_createserver_options_secureconnectionlistener):

> rejectUnauthorized <boolean> If not false the server will reject any connection which is not authorized with the list of supplied CAs. This option only has an effect if requestCert is true. Default: true.

This is not what I think of as “authentication” though, especially in the context of HTTP.

I chose `ignoreInsecure` for similarity with curl(1)’s `--insecure` option.

Something like `skipCertificateVerification` might be more obvious when reading?
Can we use default values in object destructuring? If so we could use positive wording and let `verifyCertificates: false` skip the verification (and assume `true` by default).

I’m also unsure about verify vs. validate here; HTTPie has a `--verify` option which is why I stuck to verify in the examples above.

Reply:
> I chose `ignoreInsecure` for similarity with curl(1)’s `--insecure` option.

Good thinking (though become a little wary of using cURL as an aesthetic reference... ). In that case, why not simply go with `insecure: true`? That might be a little simplistic, but seems like it should be both easy to grasp and appropriately cautionary - `ignoreInsecure` throws my brain into a loop similar to double negation...

Perhaps @cdent has an opinion here?